WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1,5 +1,6 @@
 import Combine
 import Photos
+import SwiftUI
```

Comment:
FYI: if you create a hosting controller subtype in the intro view implementation file, you would not need to import this in this file.

Reply:
I see that this is a common pattern in the codebase. I'll see if I can replicate it.

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/11011/commits/726a20ae246482474fbcb37cbf8cb01ef4534866